src/layout/NestedPicker.js

```diff
@@ -28,20 +28,29 @@ let FilteredSection = observer(
   )
 )
 FilteredSection.displayName = 'FilteredSection'
-let Section = observer(({ options, onClick, selected, Item }) => (
+let Section = observer(({ options, onClick, selected, Item = 'div' }) => (
   <div>
     {F.mapIndexed(
-      (item, key) => (
-        <Item
-          key={key}
-          onClick={() => onClick(item.value || key, item)}
-          active={selected === key}
-          disabled={selected && selected !== key}
-          hasChildren={!isField(item)}
-        >
-          {isField(item) ? item.shortLabel || item.label : _.startCase(key)}
-        </Item>
-      ),
+      (item, key) =>
+        Item === 'div' ? (
```

Comment:
Instead of doing this branching, a simpler approach is probably to just define a `DefaultItem` component that renders a div and white lists the props it passes through. This is also way more explicit about the assumptions contexture-react makes about the props it expects its components to take - special casing `div` isn't right, because that leaves it open to break for a `span` or something else. The idea is that we're saying "an `Item` component takes these props", with a default component that ignores `active` and `hasChildren` and passes the rest along to a `div`.

Also, if this is for a GreyVest scenario, I'd be surprised if the provided `Item` is actually just a div. GreyVest is the most robust theme we have - are you sure it's not using these?